### ISSUE TITLE
Anti-perilipin-1 antibody positivity

### ISSUE THREAD

**Issue by MaudDLVT** [obophenotype/human-phenotype-ontology#10385]
**Preferred term label:**
Anti-perilipin-1 antibody positivity

**Synonyms**
Anti-PLIN1 antibody positivity, anti-perilipin 1 IgG autoantibodies positivity

**Definition (free text, please give PubMed ID)**
The presence of autoantibodies in the serum that react against perilipin-1 (PLIN1). PMID: 30283460, PMID: 35771980, PMID: 35709010, PMID: 36538601

**Parent term (use hpo.jax.org/app)**
HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:79086

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-3387-1836

**Comment by pnrobinson** [comment]
@MaudDLVT 
thanks!
added as HP:6000380
